client/packages/coldchain/src/Monitoring/ListView/TemperatureLog/Toolbar.tsx

```diff
@@ -52,7 +52,7 @@ export const Toolbar: FC<{ filter: FilterController }> = () => {
             },
             {
               type: 'enum',
-              name: t('label.type'),
+              name: t('label.breach-type'),
```

Comment:
renamed the filter - just `type` by itself is different from figma and not very helpful